docs/en_US/reference/experiment_config.rst

```diff
@@ -538,7 +538,7 @@ If `trialGpuNumber`_ is less than the length of this value, only a subset will b
 This will be used as ``CUDA_VISIBLE_DEVICES`` environment variable.
 
 
-trialPrepareCommand
+trialPythonPath
 -------------------
 
 Command(s) to run before launching each trial.
```

Comment:
The path will insert to `PATH` before launching each trial.

Reply:
Done.